docs/content/latest/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/kubernetes.md

```diff
@@ -175,130 +168,122 @@ You can create a `kubeconfig` file for previously created `yugabyte-platform-uni
+- Use the **Namespace** field to specify the namespace. If provided SA has the `Cluster Admin` permissions, you are not required to complete this field. The SA used in the provided `kubeconfig` file should have access to this namespace.
+- Use **Kube Config** to upload the configuration file. If this file is available at provider level, you are not required to supply it.
 
 <img title="K8s Configuration -- zone config" alt="K8s Configuration -- zone config" class="expandable-image" src="/images/ee/k8s-setup/k8s-az-kubeconfig.png" />
 
-- `Overrides` is *optional*, if not specified Yugabyte Platform would use defaults specified inside the helm chart,
-
-- Overrides to add Service level annotations
-
-```yml
-serviceEndpoints:
-  - name: "yb-master-service"
-    type: "LoadBalancer"
-    annotations:
-      service.beta.kubernetes.io/aws-load-balancer-internal: "0.0.0.0/0"
-    app: "yb-master"
-    ports:
-      ui: "7000"
-
-  - name: "yb-tserver-service"
-    type: "LoadBalancer"
-    annotations:
-      service.beta.kubernetes.io/aws-load-balancer-internal: "0.0.0.0/0"
-    app: "yb-tserver"
-    ports:
-      ycql-port: "9042"
-      yedis-port: "6379"
-      ysql-port: "5433"
-```
-
-- Overrides to disable LoadBalancer
-
-```yml
-enableLoadBalancer: False
-```
-
-- Overrides to change the cluster domain name
+- Complete the **Overrides** field using one of the provided options. If you do not specify anything, Yugabyte Platform would use defaults specified inside the Helm chart. The following overrides are available:
+
+  - Overrides to add service-level annotations:
+
+    ```yml
+    serviceEndpoints:
+      - name: "yb-master-service"
+        type: "LoadBalancer"
+        annotations:
+          service.beta.kubernetes.io/aws-load-balancer-internal: "0.0.0.0/0"
+        app: "yb-master"
+        ports:
+          ui: "7000"
+    
+      - name: "yb-tserver-service"
+        type: "LoadBalancer"
+        annotations:
+          service.beta.kubernetes.io/aws-load-balancer-internal: "0.0.0.0/0"
+        app: "yb-tserver"
+        ports:
+          ycql-port: "9042"
+          yedis-port: "6379"
+          ysql-port: "5433"
+    ```
 
-```yml
-domainName: my.cluster
-```
+  - Overrides to disable LoadBalancer:
+  
+    ```yml
+    enableLoadBalancer: False
+    ```
+  
+  - Overrides to change the cluster domain name:
+  
+    ```yml
+    domainName: my.cluster
+    ```
+  
+  - Overrides to add annotations at StatefulSet-level:
 
-- Overrides to add annotations at StatefulSet level
+    ```yml
+    networkAnnotation:
+      annotation1: 'foo'
+      annotation2: 'bar'
+    ```
 
-```yml
-networkAnnotation:
-  annotation1: 'foo'
-  annotation2: 'bar'
-```
+  - Overrides to add custom resource allocation for YB master and TServer pods and it overrides the instance types selected in the Yugabyte universe creation flow:
+  
```

Comment:
Overrides to add custom resource allocation for YB master and TServer pods and override the instance types selected in the Yugabyte universe creation flow

Reply:
I think it should be:
Overrides to add custom resource allocation for YB master and TServer pods (this overrides the instance types selected in the Yugabyte universe creation flow):